web/blueprints/properties/__init__.py

```diff
@@ -54,7 +56,8 @@ def property_groups():
 def property_group_create():
     form = PropertyGroupForm()
     if form.validate_on_submit():
-        group = PropertyGroup(name=form.name.data)
+        group = PropertyGroup(name=form.name.data,
```

Comment:
Although not an immediate security issue: Should it be allowed to create a property group with a permission level greater than your own?

Reply:
Regarding your three comments:
I added a check for creating/editing/deleting groups in the first place, but then realized that you could just create a group on your permission level with still all properties enabled that you want.
So it would have been needed a permission classification for properties too. I then decided that "groups_change" is kind of a super permission.
However, if wanted, I could also add a classification for properties.

Reply:
Oops I missed the added `@access.require('groups_change')`, and the diff conveniently hid the already existing one on `property_group_create`, sorry. With that knowledge the whole changeset makes sense.